test/e2e_federation/upgrade.go

```diff
@@ -25,9 +25,7 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var upgradeTests = []upgrades.Test{
-	upgrades.SecretUpgradeTest(),
-}
+var upgradeTests = upgrades.GenericUpgradeTests()
```

Comment:
I'm assuming there are going to be more involved upgrade tests in the future - e.g. for ingress - and that ``upgradeTests`` will collect both the generic and non-generic tests.